components/automate-ui/src/app/entities/infra-roles/infra-role.effects.ts

```diff
@@ -48,7 +48,7 @@ export class InfraRoleEffects {
         const msg = payload.error.error;
         return new CreateNotification({
           type: Type.error,
-          message: `Could not get infra roles: ${msg || payload.error}`
+          message: `Could not get roles: ${msg || payload.error}`
```

Comment:
Would infra roles make more sense, as just roles might be confused with internal roles ?

Reply:
Good question! I considered that, but decided against it for two reasons.
1. Omitting it is more consistent with the rest of the infra pages.
2. More importantly, the context makes it clear which type of role.
That said, if you have a strong preference I am fine to change it because your point is valid.